### ISSUE TITLE
jshint option esversion not working

### ISSUE THREAD

**Issue by SHAHRUKH-KHAN** [gruntjs/grunt-contrib-jshint#254]
This is snippet from my gruntfile.js

```
jshint: {
       files: ['Gruntfile.js', 'src/**/*.js', 'test/**/*.js'],
       options: {
         globals: {
           jQuery: true,
           console: true,
           module: true,
           document: true
         },
         esnext: true, //if replaced by esversion:6, throws Bad Option
       }
     }
```

Is this not implemented or am I missing something as jshint works fine with esnext but this is deprecated.

**Comment by dgisser** [comment]
#234 related

**Comment by dgisser** [comment]
wasn't this fixed in 3b3fed2ff204db439aeabdf1c11535b2310fee64? Can we close this?

**Comment by scowalt** [comment]
From trying this just now, the esversion option works in the grunt config but doesn't work from a .jshintrc file in the root of the project directory. Not sure if this is by-design or a separate bug.
